src/kicad/board.ts

```diff
@@ -1311,6 +1311,10 @@ export class Text {
 export class FpText extends Text {
     type: "reference" | "value" | "user";
 
+    // I don't know the how locked attribute (unlocked attribute in the `At` class) affects rendering
```

Comment:
It doesn't affect rendering, at least for now, so this comment can be removed.

Reply:
It has been removed now.